app/code/Magento/Catalog/view/adminhtml/web/js/product-gallery-use-default.js

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
I believe this should be implemented via Knockout to not mix Jquery and Knockout (but I am not sure)